include/vctr/Expressions/Core/Multiply.h

```diff
@@ -78,6 +78,19 @@ class MultiplyVectors : ExpressionTemplateBase
         return dst;
     }
 
+
+    VCTR_FORCEDINLINE void evalVectorOpMultiplyAccumulate (value_type* dst) const
+    requires is::suitableForAccelerateRealFloatBinaryVectorOp<SrcAType, SrcBType, value_type, detail::dontPreferIfIppAndAccelerateAreAvailable> && is::anyVctr<SrcAType> && is::anyVctr<SrcBType>
+    {
+        Expression::Accelerate::multiplyAdd (srcA.data(), srcB.data(), dst, dst, size());
+    }
+
+    VCTR_FORCEDINLINE void evalVectorOpMultiplyAccumulate (value_type* dst) const
+    requires is::suitableForIppRealOrComplexComplexFloatBinaryVectorOp<SrcAType, SrcBType, value_type, detail::preferIfIppAndAccelerateAreAvailable> && is::anyVctr<SrcAType> && is::anyVctr<SrcBType>
```

Comment:
Is this `preferIfIppAndAccelerateAreAvailable` because `IPP::multiplyAccumulate` is faster than `Accelerate::multiplyAdd` (that has an extra destination parameter and presumably includes an extra copy step because of that)?